vm/src/obj/objtype.rs

```diff
@@ -327,6 +334,17 @@ impl PyClassRef {
             }
         }
 
+        if let Some(initter) = typ.get_super_attr("__init_subclass__") {
+            let initter = PyClassMethod::descr_get(
+                vm,
+                initter,
+                None,
+                OptionalArg::Present(typ.clone().into_object()),
+            )?;
+            let init_args = PyFuncArgs::from(kwargs);
+            vm.invoke(&initter, init_args)?;
```

Comment:
The second argument to `vm.invoke` can be any type that's `Into<PyFuncArgs>`:
```suggestion
            vm.invoke(&initter, kwargs)?;
```